tests/steps/other_v2_steps.py

```diff
@@ -97,6 +98,31 @@ def read_program(context, path):
     return read_program_binary(path)
 
 
+# Send transactions to progress block numbers on dev mode.
+def send_zero_transactions(context, txns=1):
+    sp = context.app_acl.suggested_params()
+    for _ in range(txns):
+        payment = transaction.PaymentTxn(
+            context.accounts[0],
+            sp,
+            constants.ZERO_ADDRESS,
+            random.randint(100000, 900000),
```

Comment:
Was this random payment arrived at for avoiding transaction duplication? If so, this can also be achieved by randomizing the note field. It might prove worthwhile to send the smallest amount possible.

Reply:
I played around with `randbytes` and the `note` field previously and just decided on randomizing the amount field: https://github.com/algorand/py-algorand-sdk/pull/360/commits/70c034b6405da73ac368ed36e7b77b8b2294652c#diff-9c51d98c2ed173b0192084b3c3aec41f30bfaa0ade57f240f63863602942d184L110

If you feel strongly about a particular way, I could go back to randomizing the note field instead.

Reply:
I don't feel strongly. Knowing that an exact amount is deducted each time gives a way of calculating how many such transactions were called (but perhaps in dev mode, the round number gives similar info)